blockchain/source/contracts/Migrations.sol

```diff
@@ -12,11 +12,11 @@ contract Migrations {
         owner = msg.sender;
     }
 
-    function setCompleted(uint completed) public restricted {
+    function setCompleted(uint completed) restricted {
```

Comment:
why removed visibility?

Reply:
truffle doesn't properly detect API of the migrations contract in this case

Reply:
@quasisamurai its ok, bro